x/ibc/core/24-host/validate.go

```diff
@@ -16,6 +16,10 @@ import (
 // adjusting this restriction.
 const DefaultMaxCharacterLength = 64
 
+// DefaultMaxPortCharacterLength defines the default maximum character length used
+// in validation of port identifiers.
+var DefaultMaxPortCharacterLength = 128
```

Comment:
wasmdv0.27.0 uses v3 of ibc-go.
https://github.com/cosmos/ibc-go/blob/v3.0.0/modules/core/24-host/validate.go#L21

I added DefaultMaxPortCharacterLength becase the source related to ibc has not been UPDATED in our repository.

Reply:
I added issue.
https://github.com/line/lbm-sdk/issues/585